README.md

```diff
@@ -24,6 +24,7 @@ the required compilers, libraries and build tools. Using the frozen toolchain
 allows you to avoid changing anything in your build machine to meet Scylla's
 requirements - you just need to meet the frozen toolchain's prerequisites
 (mostly, Docker or Podman being available).
+Notice that ScyllaDB maintains the current and previous releases of Scylla Open Source.
```

Comment:
That's very uninformative. We don't maintain all previous releases.

Reply:
Also, I couldn't even figure out what you tried to convey here - that ScyllaDB the company maintains the code?  If that's the intention, why did you choose to write it in the end of the seemingly-unrelated "Build Prerequisites" section,  especially when we already have a section right before it about "What is Scylla"?